packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.spec.tsx

```diff
@@ -248,4 +257,29 @@ describe('<SampleTypePropertiesPanel/>', () => {
 
         wrapper.unmount();
     });
+
+    test('premium edition with barcodes', async () => {
+        LABKEY.moduleContext = {
+            api: {
+                moduleNames: ['premium']
+            },
+        }
+        const wrapper = mount(
+            <SampleTypePropertiesPanel
+                {...BASE_PROPS}
+                model={sampleTypeModel}
+                updateModel={jest.fn}
+                onAddParentAlias={jest.fn}
+                onRemoveParentAlias={jest.fn}
+                onParentAliasChange={jest.fn}
+                onAddUniqueIdField={jest.fn}
+                parentOptions={[]}
+            />
+        );
+
+        await sleep();
+
+        expect(wrapper.find(UniqueIdBanner)).toHaveLength(1);
```

Comment:
Do we need to add the opposite case for this to check that the UniqueIdBanner doesn't show for community?